awscli/customizations/s3/subcommands.py

```diff
@@ -298,6 +298,22 @@
 }
 
 
+METADATA = {
+    'name': 'metadata', 'cli_type_name': 'map',
+    'schema': {
+        'type': 'map',
+        'key': {'type': 'string'},
+        'value': {'type': 'string'}
+    },
+    'help_text': (
+        "A map of metadata to store with the objects in S3. This will be "
+        "applied to every object which is part of this request. When copying "
+        "between two s3 locations, the metadata-directive argument will need to"
+        "be set to REPLACE."
```

Comment:
Update the docstring to remove the tidbit about needing to provide REPLACE for metadata-directive as we now autopopulate it.